docker/install.sh

```diff
@@ -33,6 +33,9 @@ fi
 
 cd ~/carma_ws
 
+echo "Installing multiple object tracking dependencies"
+sudo ./src/multiple_object_tracking/scripts/install_dependencies.sh
```

Comment:
It looks like CI using the CircleCI config is currently failing. Could you add a step for this in `.circleci/config.yml`? Looks like we could add the step after the `docker/checkout.bash` script is called [here](https://github.com/usdot-fhwa-stol/carma-platform/blob/d173d2db1be0748e2e6593a41351cdd2236d7823/.circleci/config.yml#L59).